### PR TITLE
6567 fixed lab_history amd harvest table columns and dattaypes for cdc edc report for April 2021

### PR DESCRIPTION
**Pull request by lav-patel**

### CODE REVIEW

**Comment by lav-patel** [comment]
Is this ok to merge to master ? @schandaka @nabuelrub. I will need this for CDC CDM query v10.

File: Oracle/lab_history.sql [hunk 1]
```diff
@@ -1,3 +1,4 @@
+alter session set current_schema=pcornet_cdm;
```
Comment (nabuelrub) [inline Oracle/lab_history.sql]: Is this line necessary? Can you remove it? @lav-patel

Comment (nabuelrub) [reply Oracle/lab_history.sql]: oh I see you removed it in the next commit

**Review by nabuelrub — COMMENTED** [review]
looks good to me

**Pull request merged**